state/state_test.go

```diff
@@ -2872,3 +2872,26 @@ func TestExecuteWithoutUpdatingMT(t *testing.T) {
 	assert.Equal(t, executorclientpb.RomError(1), processBatchResponse.Responses[1].Error)
 	assert.Equal(t, "0000000000000000000000000000000000000000000000000000000000000001", hex.EncodeToString(processBatchResponse.Responses[1].ReturnValue))
 }
+
+func TestStoreEvent(t *testing.T) {
```

Comment:
It is not a big deal but, shouldn't this test be in `pgstatestorage_test.go` file??

Reply:
Yes